src/BuildASub.java

```diff
@@ -5,9 +5,18 @@
     public static void main(String[] args) throws Exception {
         String bread = chooseBread();
         String meat = chooseMeat();
+        Boolean salad = addSalad ();
```

Comment:
should also be a little more meaningful, calling it `isSaladRequired` or `hasSalad` would be more informative